web/themes/custom/usagov/templates/footer-identifier.html.twig

```diff
@@ -3,7 +3,7 @@
 	<section class="usa-identifier__section usa-identifier__section--usagov" aria-label="USAGov is the Official Guide to Government Information and Services">
```

Comment:
Just an observation/question -- this line caught my eye because it has the same text as below, but in Title Case. Of course it probably sounds the same! But then I got to thinking we should also have Spanish here. And then I looked up aria-label (because this is not my strong suit) and found myself wondering why we're not using aria-labelledby, and then, why this needs to be labeled at all. Am I missing something here? 

This is all beside the point of USAGOV-692. And I know we have a tickets about translating the aria attributes in our future.